packages/sdk/src/Config.ts

```diff
@@ -228,7 +228,9 @@ export interface EthereumNetworkConfig {
 //   in Ethereum network
 export type EnvironmentId = 'polygon' | 'polygonAmoy' | 'dev2'
 
-export const DEFAULT_ENVIRONMENT: EnvironmentId = 'polygon'
+export const ENVIRONMENT_IDS: EnvironmentId[] = ['polygon', 'polygonAmoy', 'dev2']
+
+export const DEFAULT_ENVIRONMENT_ID: EnvironmentId = 'polygon'
```

Comment:
Is it so that any value added here needs to be added ton `config.schema.json` and vice versa?

Reply:
Yes